webhooks-extension/src/components/WebhookBranches/WebhookBranches.scss

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+@import "~carbon-components/scss/globals/scss/vars";
+
+#webhook-branches-modal {
+  .bx--data-table tbody td.cellText {
+    padding-left: 22.5px;
+    &[data-status] {
+      background-color: $inverse-support-01;
+      color: black;
+    }
+
+    &[data-status="Success"],
+    &[data-status="Succeeded"] {
```

Comment:
what's the difference between 'Success' and 'Succeeded'? seems odd to have both

Reply:
the reason behind this is that for some cases the PipelineRun was returning with `Success` status while other times with `Succeeded`

Reply:
something that should be raised in tektoncd/pipeline perhaps?

Reply:
probably. I double checked this with the current `PipelineRuns` I have and they are currently displaying `Succeeded`. So if @dibbles can confirm he also is getting the same then i am happy to remove `Success`

Reply:
@dibbles @AlanGreene its settled then! updating, let me know if anything else needs to be addressed